app/views/unattended/provisioning_templates/snippet/redhat_kickstart_rhsm.erb

```diff
@@ -0,0 +1,49 @@
+<%- if host_param('syspurpose_role') || host_param('syspurpose_usage') || host_param('syspurpose_sla') || host_param('syspurpose_addons') %>
+<%-
+  syspurpose_args = []
+  ['role', 'usage', 'sla'].each do |arg|
+    if (value = host_param("syspurpose_#{arg}")
+      syspurpose_args << "--#{arg}=\"#{value}\""
+    end
+  end
+  if syspurpose_args.any?
+-%>
+syspurpose <%= syspurpose_args.join(' ') %>
+<% end -%>
+<%-
+  rhsm_args = []
+  if host_param('kt_activation_keys')
+    subscription_manager_certpkg_url = subscription_manager_configuration_url(@host)
+    subscription_manager_org = @host.rhsm_organization_label
+    activation_key = host_param('kt_activation_keys')
+  else
+    subscription_manager_certpkg_url = host_param('subscription_manager_certpkg_url')
+    subscription_manager_org = host_param('subscription_manager_org')
+    activation_key = host_param('activation_key')
+  end
+
+  # kickstart only supports activation keys
+  if activation_key
+    rhsm_args << "--organization=\"#{subscription_manager_org}\""
+    rhsm_args << "--activation-key=\"#{activation_key}\""
+    # TODO add --connect-to-insights if desired
```

Comment:
Also a TODO to add URLs to point to the correct Candlepin and Pulp instances when Katello is used.

Reply:
RHEL Satellite support in Anaconda in RHEL 9.0+ it should be enough to specific just URL to the Satellite instance via `--server-hostname` - everything else, including the baseurl seems to be configured via the Satellite provisioning script that Anaconda downloads and executes.